internal/plugin/grpc_stdio.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+package plugin;
+option go_package = "plugin";
+
+import "google/protobuf/empty.proto";
+
+// GRPCStdio is a service that is automatically run by the plugin process
+// to stream any stdout/err data so that it can be mirrored on the plugin
+// host side.
+service GRPCStdio {
+  // StreamStdio returns a stream that contains all the stdout/stderr.
+  // This RPC endpoint must only be called ONCE. Once stdio data is consumed
+  // it is not sent again.
+  //
+  // Callers should connect early to prevent blocking on the plugin process.
+  rpc StreamStdio(google.protobuf.Empty) returns (stream StdioData);
+}
+
+// StdioData is a single chunk of stdout or stderr data that is streamed
+// from GRPCStdio.
+message StdioData {
+  enum Channel {
+    STDOUT = 0;
```

Comment:
Probably not a big deal, but should zero here be `UNSPECIFIED` to make it non-optional to set and follow the normal style (this convention may be dated to v2, I'm not sure)?

Reply:
I guess optionally you could just skip zero.

Reply:
I also like that, only to align stdout and stderr with their default FDs, just for some sense of superfluous consistency ;)

Reply:
I'll set a zero value here, I usually do that :)

Reply:
Done (not you do have to explicitly set a zero value or `protoc` errors)